### PR TITLE
Add a replicas field to the vineyard Runtime and delete the svc suffix in the vineyard helm chart.

### PR DESCRIPTION
**Pull request by dashanji**
### Ⅰ. Describe what this PR does

As titled.

### CODE REVIEW

File: pkg/ddc/vineyard/transform.go [hunk 1]
```diff
@@ -109,10 +109,13 @@ func (e *VineyardEngine) transformMasters(runtime *datav1alpha1.VineyardRuntime,
 
 func (e *VineyardEngine) transformWorkers(runtime *datav1alpha1.VineyardRuntime, value *Vineyard) (err error) {
 	value.Worker = Worker{}
-	if runtime.Spec.Worker.Replicas == 0 {
-		value.Worker.Replicas = 1
-	} else {
+	// respect the replicas of runtime if the replicas of worker and runtime are both specified
+	if runtime.Spec.Replicas != 0 {
```
Comment (cheyang) [inline pkg/ddc/vineyard/transform.go:113]: I think if the user doesn't specify `runtime.Spec.Replicas`, it should be 0. I don't suggest using the default value as 1.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/vineyard/transform.go (hunk 1 above)
Comment (dashanji) [reply pkg/ddc/vineyard/transform.go:113]: Ok, thanks.

**Comment by dashanji** [comment]
@cheyang Could you please take a look at this? Thanks.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**